### ISSUE TITLE
ChooseDate is not working

### ISSUE THREAD

**Issue by soyersoyer** [ManukMinasyan/vue-functional-calendar#22]
I want to programatically choose a date with 
```
this.$refs.calendar.ChooseDate('8/5/2019');
```
but it's not working.

**Comment by ManukMinasyan** [comment]
Hello @soyersoyer .
Please use the correct syntax for API.
`chooseDate('08/05/2019')`
Thank you for your response.

**Comment by soyersoyer** [comment]
Thanks for your help. When I call the ChooseDate in date picker mode, the model is not updating, the day is not selected in the view either.

**Comment by soyersoyer** [comment]
I found a workaround: modifying the model for date choosing.

**Comment by Orange289** [comment]
In my situation, the `ChooseDate` turns the calendar to the necessary month and year, but the date still remains unselected (the class doesn't change to `vfc-marked`) :( 

Here is the code fragment:
```
<FunctionalCalendar class="calendar" ref="calendar"
    :dayNames= "dayNames"
    :is-date-picker="true">
  </FunctionalCalendar>
```
```
  data () {
    return {
      date: '13/08/2019',
      dayNames: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
    }
  },

mounted() {
    this.$refs.calendar.ChooseDate(this.date);
  }

```

**Comment by ManukMinasyan** [comment]
Hello @Orange289 , please look at this example.
[https://codesandbox.io/s/vuejs-functional-calendar-component-uylhy?fontsize=14](https://codesandbox.io/s/vuejs-functional-calendar-component-uylhy?fontsize=14)

Thanks.

**Comment by Orange289** [comment]
Hello! Thank you for example! 

However, I still don't see the marked date: 
it works exactly as I described before - the ChooseDate turns the calendar to the necessary month and year, but the date still remains unselected (the class doesn't change to vfc-marked)

**Comment by Adizbek** [comment]
@Orange289 I figured out, but it's some tricky.

```vue
<FunctionalCalendar
      ....
      :marked-dates="markedDates"

      ref="calendar" v-model="calendar">
</FunctionalCalendar>
```

```js
computed: {
    markedDates () {
      const date = this.selectedDate.toDate()

      return [`${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}`]
    }
  }
```

It works, I have tested.

**Comment by BrenoHM** [comment]
Hello, in the FunctionalCalendar.vue file below line 830, this code:
this.calendar.selectedDate = `$ {newDate.getDate ()} / $ {newDate.getMonth () + 1} / $ {newDate.getFullYear ()} `;

**Comment by matteodoozy** [comment]
I have the same problem. I have a calendar1 that, when you click on a day, opens a modal with calendar2 inside. I just want to highlight the selected day on calendar1 to calendar2 using function Choosedate. It works on data, but css is not applied and the day on calendar2 is not marked with `vfc-marked` class.
I just tried the 

> @Orange289 I figured out, but it's some tricky.
> 
> ```
> <FunctionalCalendar
>       ....
>       :marked-dates="markedDates"
> 
>       ref="calendar" v-model="calendar">
> </FunctionalCalendar>
> ```
> 
> ```js
> computed: {
>     markedDates () {
>       const date = this.selectedDate.toDate()
> 
>       return [`${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}`]
>     }
>   }
> ```
> 
> It works, I have tested.

I have the same problem. I have a calendar1 that, when you click on a day, opens a modal with calendar2 inside. I just want to highlight the selected day on calendar1 to calendar2 using function Choosedate. It works on data, but css is not applied and the day on calendar2 is not marked with `vfc-marked` class.
I just tried the `marked-dates` method, but it doesn't work for me. Anyone has resolve this issue?
